tests/test_create.py

```diff
@@ -995,14 +1004,6 @@ def test_update_deps_flag_present(self):
             assert package_is_installed(prefix, 'itsdangerous')
             assert package_is_installed(prefix, 'flask')
 
-    # @pytest.mark.skipif(not on_win, reason="shortcuts only relevant on Windows")
-    # def test_shortcut_in_underscore_env_shows_message(self):
-    #     prefix = make_temp_prefix("_" + str(uuid4())[:7])
-    #     with make_temp_env(prefix=prefix):
-    #         stdout, stderr = run_command(Commands.INSTALL, prefix, "console_shortcut")
-    #         assert ("Environment name starts with underscore '_'.  "
-    #                 "Skipping menu installation." in stderr)
-
```

Comment:
Let's put this one back.  The rest of the commented out test code is fine to be gone.

Reply:
Sure. But I just want to know, why these commented code are necessary?